### PR TITLE
Test case venus

### PR DESCRIPTION
**Pull request by Domnyk**

### CODE REVIEW

**Comment by Domnyk** [comment]
Initial results:

* After 50 epochs of training AUC of test set is equal to 0.727676. Probably overfitting has occured. Around 30th epoch (see [this plot](https://drive.google.com/file/d/1-TlIB4JGtrjA2C4NoQZXBn7i3IXyg1XK/view?usp=sharing)) validation error is rising while training error continues to decline
* I have added stop condition (val_error > train_error). When applied learning stops after 28th epoch and AUC on training set is equal to 0.687380. **This is strange: I would except AUC to be higher**. Possible explanation: I stop learning after 28th epoch but weights are already updated. Perhaps I should prevent this one last update - perhaps I should use weight from 27th epoch.
